releasetool/commands/common.py

```diff
@@ -91,6 +91,19 @@ def setup_github_context(
 
     click.secho(f"Origin: {ctx.origin_repo}, Upstream: {ctx.upstream_repo}")
 
+    # Compare upstream/master with master
+    click.secho(f"> Comparing {ctx.upstream_name}/master to master.", fg="cyan")
+
+    if releasetool.git.diff(f"{ctx.upstream_name}/master", "master") == "":
```

Comment:
Would it be possible to compare the SHAs instead of the diff? I think that'll be a bit more stable.